src/testing/harness.ts

```diff
@@ -133,16 +133,19 @@ export function harness(
 		trigger(selector: string, functionSelector: string | FunctionalSelector, ...args: any[]): any {
 			_tryRender();
 			const [firstItem] = select(selector, _getRender());
-			if (firstItem) {
-				let triggerFunction: Function | undefined;
-				if (typeof functionSelector === 'string') {
-					triggerFunction = (firstItem.properties as any)[functionSelector];
-				} else {
-					triggerFunction = functionSelector(firstItem);
-				}
-				if (triggerFunction) {
-					return triggerFunction.apply(widget, args);
-				}
+
+			if (!firstItem) {
+				throw new Error(`Cannot find node with selector ${selector}`);
```

Comment:
I like this error message. It might be good to make this and the assertion template errors consistent.

https://github.com/dojo/framework/blob/4526fecf0feb91df26ad8f3a8f030f13822dff71/src/testing/assertionTemplate.ts#L36

Reply:
I don't disagree, but updating the message there is a bit more complicated. I've made the change though, we'll see how it goes 😄